### ISSUE TITLE
Losing tasks after update to 6.3.5

### ISSUE THREAD

**Issue by GarciaPL** [mpcjanssen/simpletask-android#345]
I have just updated few seconds ago Simpletask to version 6.3.5 through FDroid. It is hard to say, but I lost my tasks. I mean that they are not removed, but application is not be able to load them for some reason.

For those who might be frightened about this issue, you might found your todo.txt file under path /storage/emulated/0/data/nl.mpcjanssen.simpletask/todo.txt

**Comment by mpcjanssen** [comment]
Are you on Android M?

**Comment by mpcjanssen** [comment]
Also can you send me a logcat? I obviously don't get crash reports via FDroid

**Comment by mpcjanssen** [comment]
Did you uninstall the previous version before updating? This should only be an issue in that case. The default location of the todo file changed.

**Comment by GarciaPL** [comment]
I have Android 5.1.1. You might found logs as a attachment. I never remove previous version before installing new one. I did experiment. I uninstalled Simpletask and then install 6.3.4. Now I can see my tasks. But when I uninstalled Simpletask and then install 6.3.5 (newer one), app does not load any tasks.

What is the default location now ? Maybe there is a way just to copy todo.txt to new location.

[simpletask.txt](https://github.com/mpcjanssen/simpletask-android/files/101830/simpletask.txt)
